src/components/pages/GamePlay/GameDrawStep.js

```diff
@@ -50,11 +50,16 @@ export default function GameDrawStep(props) {
         props.updateModalStatus('draw', true);
       }
 
-      props.updateFileSubmissionData('drawings', []);
```

Comment:
Good catch on finding the problem here. The code looks good and this will make sure the user does not loose their drawing if they decide to go back. This will make sure the user experience is smooth. Good work!